build.gradle

```diff
@@ -180,6 +180,15 @@ project('segmentstore:storage:impl') {
         testCompile group: 'org.apache.hadoop', name: 'hadoop-minicluster', version: hadoopVersion, withoutLog4j
         testCompile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
 
+        //For ECS
+        /*testCompile group: 'io.findify', name: 's3mock_2.12', version: '0.2.2'
+        testCompile group: 'com.sun.jersey', name: 'jersey-json', version: '1.19'*/
+        testCompile group: 'org.gaul', name: 's3proxy', version: '1.5.2'
```

Comment:
This is used from https://github.com/andrewgaul/s3proxy. It has apache 2.0 license. We are using it just for tests as a simulator for S3 APIs.